.github/workflows/pr.yml

```diff
@@ -0,0 +1,33 @@
+on: pull_request
+name: Review
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    name: Changelog should be updated
+    strategy:
+      matrix:
+        php-versions: [ '8.3' ]
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
+
+      - name: Check that changelog has been updated.
+        run: git diff --exit-code origin/${{ github.base_ref }} -- CHANGELOG.md && exit 1 || exit 0
+
+  markdown-coding-standards:
+    name: Markdown coding standards
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Coding standards
+        run: |
+          docker run --rm --volume "$PWD:/md" itkdev/markdownlint '**/*.md'
```

Comment:
You should probably steal some markdownlint configuration and ignore rules form another project, e.g. https://github.com/itk-dev/ai-screening/.